setup.py

```diff
@@ -535,16 +535,20 @@ def add_dirs(builder, output=False):
     import pybind11
     print('PyBind11 is version ',pybind11.__version__)
     print('Looking for pybind11 header files: ')
-    for user in [True, False, None]:
-        if user is None:
+    locations = [pybind11.get_include(user=True),
+                 pybind11.get_include(user=False),
+                 '/usr/include',
```

Comment:
Maybe also include "/usr/local/include"?  Seems like that's another place things could normally be installed, so maybe worth adding that too.

Reply:
Yep, that doesn't cost much, done